### PR TITLE
[tag] added TagContainer for render tags with close button

### PR DESCRIPTION
**Pull request by ilyabrower**
## Motivation and Context
I've added wrapper for tags with clear(close) button - TagContainer

## How has this been tested?
Manually + I've added all unit tests from Tags to TagContainer

## Screenshots (if appropriate):

https://github.com/user-attachments/assets/ac65416b-fbb9-40c7-bfc4-b2d2cf84f5a7

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [X] Nice improve.

## Checklist:

- [X] My code follows the code style of this project.
- [X] I have updated the documentation accordingly or it's not required.
- [X] Unit tests are not broken.
- [X] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [X] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

**Comment by msereniti** [comment]
I can't fully get how to use the new api. Are going to update documentation?

**Comment by ilyabrower** [comment]
> I can't fully get how to use the new api. Are going to update documentation?

I've updated docs

File: semcore/tag/src/index.d.ts [hunk 1]
```diff
@@ -83,8 +83,21 @@ export type TagTextProps = BoxProps & KeyboardFocusProps & {};
 declare const Tag: Intergalactic.Component<'div', TagProps, TagContext> & {
   Text: Intergalactic.Component<'div', TagTextProps>;
   Addon: Intergalactic.Component<'div', TagAddonProps>;
+  /**
+   * @deprecated For Tags with close button, you should use
```
Comment (msereniti) [inline semcore/tag/src/index.d.ts:87]: Let's just keep a link to the full example here

Comment (ilyabrower) [reply semcore/tag/src/index.d.ts:87]: added link to the example in documentation

**Review by msereniti — APPROVED** [review]

**Review by j-mnizhek — APPROVED** [review]
Pushed small commit with styles' updates for Tag.
Will update design documentation a little bit later with other PR.

**Pull request merged**